dataflow/src/org/checkerframework/dataflow/analysis/Store.java

```diff
@@ -65,14 +65,16 @@
      *
      * <ul>
      *   <li>Does not change {@code this}.
-     *   <li>Does not change {@code other}.
+     *   <li>Does not change {@code previous}.
      *   <li>Returns a fresh object which is not aliased yet.
      *   <li>Returns an object of the same (dynamic) type as {@code this}, even if the signature is
      *       more permissive.
      *   <li>Is commutative.
      * </ul>
+     *
+     * @param previous must be the previous store
      */
-    S widenUpperBound(S other);
+    S widenUpperBound(S previous);
```

Comment:
For consistency, should this be renamed to `widenedUpperBound`?
Add a reference to the other `widenedUpperBound` and explain difference in naming?